lucille-core/src/test/java/com/kmwllc/lucille/core/FileToFileIngestIT.java

```diff
@@ -15,6 +15,7 @@ public void fullIT() throws Exception {
     File f = null;
     try {
       Config config = ConfigFactory.parseFile(new File("src/test/resources/FileToFileIngestIT/file-to-file-example.conf"));
+      new File("output/").mkdir();
```

Comment:
pls. call the directory `tempIntegrationTestOutput`
pls. delete the directory after the test